src/sort.c

```diff
@@ -580,7 +580,7 @@ void sortCommandGeneric(client *c, int readonly) {
             }
         }
         if (outputlen) {
-            listTypeTryConversion(sobj,OBJ_ENCODING_LISTPACK,NULL,NULL);
+            listTypeTryConversionForShrinking(sobj,NULL,NULL);
```

Comment:
let's comment that we created it as a quicklist, so now we attempt to shrink it to listpack if it doesn't have a lot of data.
aren't we suppose to call listTypeTryConvertQuicklist(shrinking=0) here? like we do on rdb.c when we're done blindly populating a quicklist? i.e. created a quicklist without knowing it if'll be required.

Reply:
Yes, it's same as RDB loading.

Reply:
Also fixed in [`6f1b697` (#11303)](https://github.com/redis/redis/pull/11303/commits/6f1b69710b4471f9441ed99f5211f311f0d770b1)